rails/benchmarks/bm_scaffold_create.rb

```diff
@@ -4,13 +4,6 @@
 require 'active_record'
 require_relative 'support/benchmark_rails'
 
-ENV['RAILS_ENV'] = 'production'
-
-class NullLogger < Logger
-  def initialize(*args);end
-  def add(*args, &block);end
-end
```

Comment:
I've used NullLogger here to avoid artefacts after benchmark run. If we cleanup, there is no problem to use a classic Rails log